### PR TITLE
OSDOCS-8022: Reorganizing Network Observability in the topicmap

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.11+
Issue:

https://issues.redhat.com/browse/OSDOCS-8022
Link to docs preview:

Top of Network Observability begins here now: https://65770--docspreview.netlify.app/openshift-enterprise/latest/network_observability/network-observability-operator-release-notes

comp to current location in the docs:
https://docs.openshift.com/container-platform/4.13/networking/network_observability/network-observability-operator-release-notes.html
QE review:
- [X] QE has approved this change.

Additional information:

This update reorganizes the Network Observability content to be situated under Monitoring instead of in the Networking docset.

### CODE REVIEW

File: network_observability/network-observability-operator-release-notes.adoc
Comment (jotak) [inline network_observability/network-observability-operator-release-notes.adoc:44]: typo: doubled "and"

**Comment by jotak** [comment]
Just the typo, else LGTM!

**Review by skopacz1 — APPROVED** [review]
This is a really nice reorg, awesome stuff! 

Since this is technically adding a top-level directory/book, I just want to check - did you run this change by @kalexand-rh or anyone else? I believe there may be some tooling-side work needed to accommodate a new top-level directory, so I just want to be sure that's in place before this merges.

**Comment by skrthomas** [comment]
Hey @kalexand-rh ! @ahardin-rh and I discussed this change already, but as @skopacz1 mentioned I may also need to run this by you. @Deepthidharwar approved this change in a Slack discussion, and I've added her to this PR to approve here as well. I'm not _sure_ if I need PM approval or not but figured I would ask anyways.

**Comment by briandooley** [comment]
@skrthomas I ran this by the OBS docs team and we are all agreed that this is a good idea. LGTM.

**Comment by mcurry-rh** [comment]
LGTM -- Approved.

**Review by stleerh — COMMENTED** [review]
I like the new home! Since the old link will not automatically redirect you to the new link, do you know what the new link will be?  Some blogs will need to be updated, unless we can handle the redirection (although it's still good to update the links).

**Comment by kalexand-rh** [comment]
The new release notes link should be https://docs.openshift.com/container-platform/4.13/network_observability/network-observability-operator-release-notes.html

@skrthomas, check in with @bscott-rh if you want to set up a redirect.

**Comment by skrthomas** [comment]
@kalexand-rh thanks for the heads up about that. I think I'll need to open up individual PRs against 4.11-4.14 release notes and make the change there. And I'll also check with Ben about redirects, I definitely would like to set that up just bc there are engineering blogs that link to Network Observability docs too.

**Pull request merged**
